data/live_response.yaml

```diff
@@ -0,0 +1,165 @@
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - JavaCacheFiles
+    - WindowsAMCacheHveFile
+    - WindowsCIMRepositoryFiles
+    - WindowsCrashDumps
+    - WindowsPrefetchFiles
+    - WindowsRecentFileCacheBCF
+    - WindowsStartupInfo
+    - WindowsSuperFetchFiles
+    - WindowsSystemResourceUsageMonitorDatabaseFile
+    - WMICCMRUA
+supported_os: [Windows]
+---
+name: LiveResponseExternalMedia
+doc: Group of external media data or events related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - WindowsSetupApiLogs
+supported_os: [Windows]
+---
+name: LiveResponseFileSystem
+doc: Group of file system related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - NTFSLogFile
+    - NTFSMFTFiles
+supported_os: [Windows]
+---
+name: LiveResponseHistoryFiles
+doc: Group of history files related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - ShellConfigurationFile
+    - ShellHistoryFile
+    - WindowsPowerShellHistory
+supported_os: [Windows]
+---
+name: LiveResponseInteractiveActivity
+doc: Group of interactive user activity related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - MicrosoftOfficeAutosave
+    - MicrosoftOfficeMRU
+    - WindowsActivitiesCacheDatabase
+    - WindowsRDPClientBitmapCache
+    - WindowsRecycleBinMetadata
+    - WindowsSearchDatabase
+    - WindowsUserAutomaticDestinationsJumpLists
+    - WindowsUserCustomDestinationsJumpLists
+    - WindowsUserRecentFiles
+supported_os: [Windows]
+---
+name: LiveResponseNetwork
+doc: Group of network related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - WindowsFirewallLogFile
+    - WindowsHostsFiles
+supported_os: [Windows]
+---
+name: LiveResponsePersistence
+doc: Group of persistence mechanism related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - WMIEnumerateASEC
+    - WMIEnumerateCLEC
+    - WindowsApplicationCompatibilityInstalledShimDatabases
+    - WindowsAutoexecBat
+    - WindowsAutorun
+    - WindowsBITSQueueManagerDatabases
+    - WindowsGroupPolicyScripts
+    - WindowsPowerShellDefaultProfiles
+    - WindowsScheduledTasks
+    - WindowsStartupFolders
+    - WindowsWinstart
+supported_os: [Windows]
+---
+name: LiveResponseSecurityAgents
+doc: Group of endpoint detection and response related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - EsetAVQuarantine
+    - MicrosoftAVLogs
+    - MicrosoftAVQuarantine
+    - SophosAVLogs
+    - SophosAVQuarantine
+    - SymantecAVLogs
+    - SymantecAVQuarantine
+supported_os: [Windows]
+---
+name: LiveResponseSystemConfiguration
+doc: Group of configuration files related artifacts.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - WindowsRegistryFilesAndTransactionLogs
+    - WindowsSystemRegistryFilesAndTransactionLogsBackup
+supported_os: [Windows]
+---
+name: LiveResponseSystemLogs
```

Comment:
This looks very Windows centric.

Reply:
The plan is for this to evolve to logs from other platforms.